### ISSUE TITLE
Invalid formating with anonymous class

### ISSUE THREAD

**Issue by arvitaly** [palantir/tslint#3209]
### Bug Report

- __TSLint version__: ^5.7.0
- __TypeScript version__: 2.5.2
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
const a = class {
    constructor() {
        public her() {
        console.log(111);
    }
}
};
```

with `tslint.json` configuration:

```json
{
    "extends": [
        "tslint:latest"
    ]
}
```

#### Actual behavior
Error:
[tslint] block is empty (no-empty)
[tslint] members are not aligned (align)
[ts] Declaration or statement expected.
![capture](https://user-images.githubusercontent.com/2562855/30239103-d4cd0aae-955d-11e7-9dde-6ec7912f2215.PNG)
#### Expected behavior
No error

**Comment by ajafff** [comment]
You can't declare methods inside the constructor.
That's why you get this error: `[ts] Declaration or statement expected.`

You should make sure your sources have no syntax or grammar errors and compile correctly *before* using the linter. Otherwise the linting errors don't make sense as you just experienced.

**Comment by arvitaly** [comment]
Sorry, thank you!
